src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -239,6 +247,38 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return _defaultAppearanceViewModel;
     }
 
+    bool ProfileViewModel::HasUnfocusedAppearance()
+    {
+        return _profile.HasUnfocusedAppearance();
+    }
+
+    void ProfileViewModel::CreateUnfocusedAppearance(const Windows::Foundation::Collections::IMapView<hstring, Model::ColorScheme>& schemes,
+                                                     const IHostedInWindow& windowRoot)
+    {
+        _profile.CreateUnfocusedAppearance();
+
+        _unfocusedAppearanceViewModel = winrt::make<implementation::AppearanceViewModel>(_profile.UnfocusedAppearance().try_as<AppearanceConfig>());
+        _unfocusedAppearanceViewModel.Schemes(schemes);
+        _unfocusedAppearanceViewModel.WindowRoot(windowRoot);
+
+        _NotifyChanges(L"UnfocusedAppearance");
+        _NotifyChanges(L"HasUnfocusedAppearance");
+    }
+
+    void ProfileViewModel::DeleteUnfocusedAppearance()
+    {
+        _profile.DeleteUnfocusedAppearance();
+
+        _unfocusedAppearanceViewModel = nullptr;
+
+        _NotifyChanges(L"HasUnfocusedAppearance");
```

Comment:
Why do you not also need to notify `UnfocusedAppearance` here?

Reply:
The control's visibility gets set to false from the `HasUnfocusedAppearance` notification, so XAML never even tries to access the `UnfocusedAppearance`